### ISSUE TITLE
Composer installation fails because of invalid url

### ISSUE THREAD

**Issue by oskari** [samayo/bulletproof#25]
vagrant@vagrant-ubuntu-trusty-64:/vagrant$ php composer.phar require "bullet-proof/image-uploader"                                                                                     
Using version ~1.4 for bullet-proof/image-uploader                                                                                                                                     
./composer.json has been updated                                                                                                                                                       
Loading composer repositories with package information                                                                                                                                 
Updating dependencies (including require-dev)                                                                                                                                          
  - Installing bullet-proof/image-uploader (1.4)                                                                                                                                       
    Downloading: connection...    Failed to download bullet-proof/image-uploader from dist: The "https://api.github.com/repos/bivoc/bulletproof/zipball/fab4fca10bb7c341767904d60d37297
5eecfba16" file could not be downloaded (HTTP/1.1 404 Not Found)                                                                                                                       
    Now trying to download from source

**Comment by samayo** [comment]
I can see 'bivoc' inside the link, which was my previous githib username. Have you tried replacing it with 'samayo'?

**Comment by oskari** [comment]
Yes, the problem is that you've changed your github username, but as I'm not entering the URL anywhere myself, I believe this is a problem on the package you've submitted to https://packagist.org/
As you see on https://packagist.org/packages/bullet-proof/image-uploader the 'Canonical' URL is still pointing to wrong place. Now I'm not composer expert, but could this lead to composer trying to access the download from wrong url?
